neutrons/SNAPRed
# Raw vanadium correction

**Description of work**

Implements a script to perform an unfocused normalization correction.  This is sometimes called a "raw vanadium correction," but it could use many other types of materials.

The work implements a [prototype script](https://github.com/neutrons/prototype_SNAPRed_scripts/blob/main/ewm780/raw_vanadium%20(1).py) from @mguthriem into SNAPRed.  

The main change is that this is not limited to the pre-defined vanadium cylinder, but can rather take any sample defined through the CalibrantSample DAO, and use mantid algorithms to set this as the sample.

**To test:**

Firstly, run the unit tests.

Within workbench, create reduction ingredients and a calibrant sample.

Setup the algorithm, and ensure the output has been properly corrected.

**Link to EWM item**

Work for [EWM780](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20%28Change%20Management%29#action=com.ibm.team.workitem.viewWorkItem&id=780)
